src/window.rs

```diff
@@ -1379,3 +1376,19 @@ impl Default for UserAttentionType {
         UserAttentionType::Informational
     }
 }
+
+/// ## Platform-specific
+///
+/// - **iOS / Android / Web / Wayland:** Unsupported.
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+pub enum WindowLevel {
```

Comment:
Should add a bit more documentation what exactly is meant with `WindowLevel` and how it affects window ordering in the Z-hierarchy